_topics/en/soft-fork-activation.md

```diff
@@ -0,0 +1,603 @@
+  header version field as a bitfield.  Bits in the field were used for
+  signaling only---never to reject a block as invalid---and they could
+  be set in parallel.  Measurements were taken only once every 2,016
+  blocks, minimizing the window during which a small percentage of
+  hashrate would need to be lucky to project a false impression of 95%
+  signaling.  Finally, when the 95% signaling threshold was reached,
+  there would be a 2,016 block (roughly two week) "lock in" period until
+  activation occurred to allow people to prepare.  If the signaling
+  threshold wasn't reached by a timeout date, the attempt ended,
+  allowing the unused code to be removed from later node software
+  releases.
+
+  This activation method was first used with the soft fork that added
+  [BIP68][] consensus-enforced sequence numbers, [BIP112][]
+  `OP_CHECKSEQUENCEVERIFY`, and [BIP113][] nLockTime enforcement by
+  median time past.  The fork quickly made it to the locked-in phase,
+  and then proceeded automatically to activation.
+
+  #### [2016-7] BIP9, BIP148, and BIP91: the BIP141/143 segwit activation
+
+  The segwit soft fork was released with [BIP9][] activation
+  parameters.  A few miners quickly began signaling readiness but support
+  peaked far below the 95% threshold.  Some Bitcoin users felt
+  miners were illegitimately delaying a useful new feature and went on
+  to develop the mandatory activation that became [BIP148][].  The final
+  form of BIP148 specified rejecting any
+  blocks that didn't signal support for segwit starting on a
+  certain date.
+
+  Software implementing BIP148 created the risk of a consensus failure.  If miners failed to
+  signal support for segwit, and most users continued to consider those
+  miners' blocks as valid, users of BIP148 might accept bitcoins that
+  would be invalid from the perspective of most other Bitcoin users.  Alternatively, if
+  most users of Bitcoin followed the BIP148 rules---but miners still
+  produced many BIP148-invalid blocks---those users who didn't enforce
+  BIP148 could accept bitcoins that the BIP148 users would consider
+  invalid.  Only if all users followed the same rules, or most mining
+  hashrate followed the BIP148 rules, would safety be assured.
+
+  One way to reduce this risk was to give more users time to upgrade to
+  a node that would mandate activation of segwit.  BIP148 couldn't do
+  this since its goal of triggering the existing BIP9 segwit deployment
+  meant it had to force miner signaling to start long before segwit's
+  BIP9 timeout date.  As an alternative in case BIP148 failed to gain
+  sufficient support, [BIP149][] was proposed to give users another year
+  to upgrade.  BIP149 never gained much public support, but it did
+  result in the creation of [BIP8][], which would see additional
+  discussion in subsequent years.
+
+  Around the time BIP148 began receiving significant public support,
```

Comment:
No, NYA/BIP91 were several months later, well after it became clear BIP148 would succeed. Its only purpose was face saving and attempting to tie a hardfork to it.